.github/workflows/e2e.yml

```diff
@@ -22,8 +22,9 @@ jobs:
       with:
         java-version: 11
         distribution: 'zulu'
+    - uses: gradle/gradle-build-action@v2
     - name: Run sanity check
-      run: ./gradlew --scan --stacktrace test
+      run: ./gradlew --stacktrace test
```

Comment:
Why did you remove the build scan? It is useful (among others) to get the detailed test failure from the CI build?

Reply:
Ooops, I missed there is no `publishAlways` in `buildScan`.

Reply:
3f2505745c3806e2e8f0f31f82410775a87b22c5